taprpc/universerpc/universe.yaml

```diff
@@ -37,6 +37,13 @@ http:
         - post: "/v1/taproot-assets/universe/proofs/group-key/{key.id.group_key_str}/{key.leaf_key.op.hash_str}/{key.leaf_key.op.index}/{key.leaf_key.script_key_str}"
           body: "*"
 
+    - selector: universerpc.Universe.PushProof
```

Comment:
I probably would've put the "verb" of the action a bit earlier. So for example `/v1/taproot-assets/universe/proofs/push/asset-id/.....` and `/v1/taproot-assets/universe/proofs/push/group-key/.....`.

Reply:
That was my first choice but I was undecided if that was too similar to Query and Insert. IIUC the `body` having a different name & structure should prevent any mixups as well as the path.

Reply:
Looks like this doesn't explicitly include the universe server ID. Will the REST mapping just make this zero?

Reply:
There's still the body of the REST request that takes any variable that wasn't specified in the URL. And since the universe server ID is optional, it makes sense to have it there IMO.